aikau/src/main/resources/alfresco/documentlibrary/AlfDocumentList.js

```diff
@@ -139,6 +139,11 @@ define(["dojo/_base/declare",
       onPathChanged: function alfresco_documentlibrary_AlfDocumentList__onPathChanged(payload) {
          if (payload.path)
          {
+            if (this.useInfiniteScroll === true)
+            {
+               this.clearViews();
+            }
+
```

Comment:
Seems unlikely it could be any other truthy value without someone wanting infinite scroll active. Would be less wordy to just have `this.useInfiniteScroll && this.clearViews();`